qdrant-landing/content/documentation/private-cloud/changelog.md

```diff
@@ -5,6 +5,12 @@ weight: 5
 
 # Changelog
 
+## 1.5.0
+
+Added support for P2P TLS configuration
+Faster node removal on scale down
+Various performance improvements and stability fixes
```

Comment:
Aren't they supposed to be list? I believe you forgot to add `-` or `*` at the beginning.